var/spack/repos/builtin/packages/rocblas/package.py

```diff
@@ -105,7 +107,8 @@ def check(self):
         ('@4.5.0',  '0f6a6d1557868d6d563cb1edf167c32c2e34fda0'),
         ('@4.5.2',  '0f6a6d1557868d6d563cb1edf167c32c2e34fda0'),
         ('@5.0.0',  '75b9aefe5981d85d1df32ddcebf32dab52bfdabd'),
-        ('@5.0.2',  '75b9aefe5981d85d1df32ddcebf32dab52bfdabd')
+        ('@5.0.2',  '75b9aefe5981d85d1df32ddcebf32dab52bfdabd'),
+        ('@5.1.0',  'ea38f8661281a37cd81c96cc07868e3f07d2c4da')
```

Comment:
Curious: Why use commits that don't (necessarily) line up with the commits for the associated releases?

Reply:
@srekolam is using the commit from [tensile_tag.txt](https://github.com/ROCmSoftwarePlatform/rocBLAS/blob/rocm-5.1.0/tensile_tag.txt), which specifies the version of Tensile that is downloaded and used as part of the normal rocBLAS build. For releases, tensile_tag.txt is supposed to be the same as the version of Tensile that is tagged, but it seems that our process is to update the change log as the very last step before tagging a release. Thus, tensile_tag.txt ends up slightly out-of-date.

It should be fine to use either the version tagged for release or the version from tensile_tag.txt. IMO, it's a minor process wart that they're different. They're intended to be identical.